docs/Frequently Asked Questions/Can I Integrate SDL into a React Native App/index.md

```diff
@@ -0,0 +1,231 @@
+}];
+```
+
+##### Swift
+```swift
+let softButton = SDLSoftButtonObject(name: "Button", state: SDLSoftButtonState(stateName: "State", text: "Weather", artwork: nil), handler: { (buttonPress, butonEvent) in
+    guard buttonPress == nil else { return }
+    
+    let managers = ["sdlManager":self.sdlManager]
+    NotificationCenter.default.post(name: NSNotification.Name(rawValue: "ReactNotificationGetWeather"), object: nil, userInfo: managers)
+})
+
+```
+Create the new class and add a listener for the notificaiton and all required methods for sending an event.
+
+##### Objective-C
+###### SDLEventEmitter.h
+```objc
+#import <React/RCTEventEmitter.h>
+#import <React/RCTBridgeModule.h>
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface SDLEventEmitter : RCTEventEmitter
+
+@end
+
+NS_ASSUME_NONNULL_END
+```
+###### SDLEventEmitter.m
+```objc
+#import "SDLEventEmitter.h"
+#import "ProxyManager.h"
+#import <React/RCTConvert.h>
+#import <SmartDeviceLink/SmartDeviceLink.h>
+
+@implementation SDLEventEmitter
+
+RCT_EXPORT_MODULE()
+
+- (instancetype)init {
+    self = [super init];
+    [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(getWeatherNotification:) name:@"ReactNotificationGetWeather" object:nil];
+
+    return self;
+}
+
+- (NSArray<NSString *> *)supportedEvents {
+    return @[@"GetWeather"];
+}
+
+- (void)getWeatherNotification:(NSNotification *)notification {
+    if(self.sdlManager == nil) {
+        self.sdlManager = notification.userInfo[@"sdlManager"];
+    }
+    [self sendEventWithName:@"GetWeather" body:@{@"type": @"weatherType"}];
+}
+
+@end
+```
+##### Swift
+```swift
+@objc(SDLEventEmitter)
+class SDLEventEmitter: RCTEventEmitter {
+
+override init() {
+    NotificationCenter.default.addObserver(self, selector: #selector(getWeatherNotification(_:)), name: "ReactNotificationGetWeather", object: nil)
+    super.init()
+}
+
+@objc func getWeatherNotification(notification: Notification) {
+    if self.sdlManger == nil {
+        self.sdlManager = notification.userInfo["sdlManager"]
+    }
+
+    sendEvent(withName: "GetWeather", body: ["type": "weatherType"])
+}
+
+override func supportedEvents() -> [String]! {
+    return ["GetWeather"]
+}
+
+}
+```
+Make sure you add `#import "React/RCTEventEmitter.h"` to the apps bridging header before moving forward if you making a Swift application.
+
+Now you need to create the Objective-C bridging class for `SDLEventEmitter` and add the proper `RCT_EXTERN_METHOD` wrapper. Swift ONLY.
+
+```objc
+#import "React/RCTBridgeModule.h"
+#import "React/RCTEventEmitter.h"
+@interface RCT_EXTERN_MODULE(SDLEventEmitter, RCTEventEmitter)
+
+RCT_EXTERN_METHOD(weather:(weather: (id)dict))
+
+@end
+
+```
+
+The above example will then call into JavaScript with an event type `GetWeather`.  You will need to create a  `NativeEventEmitter` with your EventEmitter module and add a listener.
```

Comment:
Remove references to weather. What is `NativeEventEmitter`? A JavaScript class? This is unclear.